specifyweb/frontend/js_src/lib/components/SchemaConfig/Fields.tsx

```diff
@@ -24,14 +24,23 @@ export function SchemaConfigFields({
   readonly onChange: (index: number) => void;
 }): JSX.Element {
   const id = useId('schema-fields');
-  const sortedItems = Object.values(items ?? []).sort(
-    sortFunction(({ name }) => name)
-  );
+  const [isHiddenFirst, setIsHiddenFirst] = React.useState(false);
```

Comment:
1. store this in local storage (see other examples in specify for how to do it)
2. default should be true